### DISCUSSION TITLE
Sympa 6.2.56 released

### DISCUSSION THREAD

**Discussion by ikedas** [sympa-community/sympa#1613] [Team Posts]
Done!

**Comment by racke** [comment]
Thanks a lot

**Comment by ikedas** [comment]
I pushed new configure options `--disable-setuid_fcgi` and `--disable-setuid_queue`.  The former pervents installing FastCGI wrappers and the latter prevents setting setuid bit on `*queue` programs.

Configuring web interface with separate wwsympa process won't use FastCGI wrappers, and Postfix virtual domain setting (with transport maps) may not need setuid bit of `*queue`.  New options may be useful on particular environments not requiring these setuid programs.
